tidb-cloud/serverless-driver.md

```diff
@@ -281,27 +281,33 @@ The type mapping between TiDB Serverless and Javascript is as follows:
 | DECIMAL              | string          |
 | CHAR                 | string          |
 | VARCHAR              | string          |
-| BINARY               | string          |
-| VARBINARY            | string          |
+| BINARY               | Uint8Array      |
+| VARBINARY            | Uint8Array      |
 | TINYTEXT             | string          |
 | TEXT                 | string          |
 | MEDIUMTEXT           | string          |
 | LONGTEXT             | string          |
-| TINYBLOB             | string          |
-| BLOB                 | string          |
-| MEDIUMBLOB           | string          |
-| LONGBLOB             | string          |
+| TINYBLOB             | Uint8Array      |
+| BLOB                 | Uint8Array      |
+| MEDIUMBLOB           | Uint8Array      |
+| LONGBLOB             | Uint8Array      |
 | DATE                 | string          |
 | TIME                 | string          |
 | DATETIME             | string          |
 | TIMESTAMP            | string          |
 | ENUM                 | string          |
 | SET                  | string          |
-| BIT                  | string          |
+| BIT                  | Uint8Array      |
 | JSON                 | object          |
 | NULL                 | null            |
 | Others               | string          |
 
+> **Note:**
+>
+> Please use utf8mb4 collation in TiDB Serverless(default is utf8mb4) for the type convert to Javascript string. Because we will use the `utf8` encoding to decode them to string. 
+
+- _Changed in v0.1.0_: The `BINARY`、`VARBINARY`、`TINYBLOB`、`BLOB`、`MEDIUMBLOB`、`LONGBLOB` and `BIT` types are now returned as a `Uint8Array` instead of a `string`.
```

Comment:
```suggestion
> **Note:**
>
> TiDB Cloud serverless driver data type mapping changes:
>  _Changed in v0.1.0_: The `BINARY`、`VARBINARY`、`TINYBLOB`、`BLOB`、`MEDIUMBLOB`、`LONGBLOB` and `BIT` types are now returned as a `Uint8Array` instead of a `string`.
```